packages/core/src/part.js

```diff
@@ -357,20 +357,68 @@ Part.prototype.generateTransform = function(transforms) {
   }
 }
 
-/** Chainable way to set the grain property */
-Part.prototype.setGrain = function (grain = 90) {
-  this.attributes.set('data-grain', grain)
+/** Chainable way to add the cut info */
+Part.prototype.addCut = function (cut=2, material='fabric', identical=false) {
+  if (cut === false) {
+    if (this.cut.materials[material]) delete this.cut.materials[material]
+    else this.context.raise.error(`Tried to remove a material that is not set`)
+    return this
+  }
+  if (typeof this.cut.materials[material] === 'undefined') this.cut.materials[material] = {}
+  if (typeof cut !== 'number') {
+    this.context.raise.error(`Tried to set cut to a value that is not a number`)
+    return this
+  }
```

Comment:
We are permitting customers to set `cut` to negative numbers and floats? I am not sure how `cut` will be used elsewhere in the code.

Reply:
Good catch. Cut needs to be positive

Reply:
I've checked for an int, but not positive. I've also allowed zero. Because some patterns have parts that are not rendered by default. The user can however choose to render them. So I feel it makes sense to set `cut=0` on these, so I've allowed that.